data/xml/1993.eamt.xml

```diff
@@ -0,0 +1,96 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<collection id="1993.eamt">
+  <volume id="1" ingest-date="2020-08-20">
+    <meta>
+      <booktitle>Third International EAMT Workshop: Machine Translation and the Lexicon</booktitle>
+      <publisher>Springer</publisher>
+      <address>Heidelberg, Germany</address>
+      <month>April 26–28</month>
+      <year>1993</year>
+      <editor><first>Petra</first><last>Steffens</last></editor>
+      <isbn>3-540-59040-4</isbn>
+    </meta>
+    <paper id="1">
```

Comment:
Add introduction? would be consistent with other years.
https://link.springer.com/book/10.1007/3-540-59040-4

Reply:
Ah, good catch. Thank you for your careful reviews!

Reply:
Comparing that link to that of the MT Archive, it seems we are also missing the paper [Machine Translation and the Lexicon](https://link.springer.com/book/10.1007/3-540-59040-4). I'll add it.